geometrical_calculator_kuba.py

```diff
@@ -0,0 +1,393 @@
+    def calculate_area(self):
+        return 1/2 * self.b * self.h
+    
+    def calculate_circumference(self):
+        return self.a + self.b + self.c
+      
+class Square:
+    
+    def __init__(self , a):
+        self.a = a
+        
+    def calculate_area(self):
+        return self.a ** 2
+    
+    def calculate_circumference(self):
+        return 4 * self.a
+        
+class Rectangle:
+    
+    def __init__(self , a , b):
+        self.a = a
+        self.b = b
+    
+    def calculate_area(self):
+        return self.a * self.b
+    
+    def caluclate_circumference(self):
+        return (2 * self.a) + (2 * self.b)
+    
+class Rhombus:
+    
+    def __init__(self , a , h):
+        self.a = a
+        self.h = h
+    
+    def calculate_area(self):
+        return self.a * self.h
+    
+    def calculate_circumference(self):
+        return 4 * self.a
+    
+class RegularPentagon:
+    
+    def __init__(self , a):
+        self.a = a
+    
+    def calculate_area(self):
+        return 0.25 * math.sqrt(5 * (5 + 2 * math.sqrt(5))) * self.a ** 2
+    
+    def calculate_circumference(self):
+        return 5 * self.a
+    
+class RegularHexagon:
+    
+    def __init__(self , a):
+        self.a = a
+    
+    def calculate_area(self):
+        return (3 * math.sqrt(3) / 2) * self.a ** 2
+    
+    def calculate_circumference(self):
+        return 6 * self.a
+    
+class RectangularPrism:
+    
+    def __init__(self , a , b , c):
+        self.a = a
+        self.b = b
+        self.c = c
+    
+    def calcuclate_area(self):
+        return 2 * (self.a * self.b) + 2 * (self.a * self.c) + 2 * (self.b * self.c)
+    
+    def calculate_circumference(self):
+        return (4 * self.a) + (4 * self.b) + (4 * self.c)
+    
+    
+def circle():
+    while True:
+        try:
+            print("To calculate THE AREA of a circle (π * r^2) - press '1'")
+            print("To calculate THE CIRCUMFERENCE of a circle (2 * π * r) - press '2'")
+            choose = input("Choose operation...")
+            if choose == '1':
+                r = float(input("Enter the radius of the circle: "))
+                circle_obj = Circle(r)
+                area = circle_obj.calculate_area()
+                geometrical_calculator_history.append((f"Circle area with radius {r}", area))
+                print("The AREA of the circle is:", area)
+                break
+            elif choose == '2':
+                r = float(input("Enter the radius of the circle: "))
+                circle_obj = Circle(r)
+                circumference = circle_obj.calculate_circumference()
+                geometrical_calculator_history.append((f"Circle circumference with radius {r}", circumference))
+                print("The CIRCUMFERENCE of the circle is:", circumference)
+        except ValueError:
+            print("Invalid choice. Please enter '1' for area or '2' for circumference.")
+        else:
+            break       
+
+
+
+
+def triangle():
```

Comment:
Czyli tak jak wyżej pisałem, rozbijamy to wszystko na np. 

`def input_triangle_sides():`
`def calculate_triangle_area():`
`def calculate_triangle_circumference():`

itd